### DISCUSSION TITLE
Expand apt package / make upgrading with Debian package user-friendly

### DISCUSSION THREAD

**Discussion by WilliamDEdwards** [meilisearch/product#717] [Feedback & Feature Proposal]
Currently, the apt package provides only the Meilisearch binary. It should cover all currently manually steps in the [Running Meilisearch in production](https://www.meilisearch.com/docs/learn/cookbooks/running_production) document (system user, config, etc.)

This would also allow upgrades to become user-friendly. Currently, when upgrading the Meilisearch package, one must manually import data as instructed by the [Update to the latest Meilisearch version](https://www.meilisearch.com/docs/learn/update_and_migration/updating) document. This process could be automated so that `apt upgrade` 'just works': create dump (in `preinst`), stop Meilisearch (requires shipping systemd unit), replace binary, start Meilisearch (requires shipping systemd unit), import dump (in `postinst`).

**Comment by macraig** [comment]
Hi @WilliamDEdwards , thanks for taking the time to leave your feedback. We're currently focusing on other product areas, but we'll keep this discussion open for other users to upvote and comment.
